src/main/charts/confluence/templates/NOTES.txt

```diff
@@ -51,5 +51,11 @@ Get the {{ title .Chart.Name }} URL by running these commands in the same shell:
 ######            Data will be lost when the pod is terminated.             #####
 #################################################################################
 {{- end }}
+{{ if ( and .Values.synchrony.enabled ( not ( or .Values.volumes.synchronyHome.persistentVolumeClaim.create .Values.volumes.synchronyHome.customVolume ))) }}
```

Comment:
If no persistent volume is specified, we write a warning. This is done only if synchrony is enabled